schema.graphql

```diff
@@ -271,6 +271,7 @@ type StateChannel @entity {
 
   indexer: String!
   consumer: String!
+  agent: String!
```

Comment:
[suggestion] agent can be optional, if the channel is open by the consumer host contract, the agent will be `c_host_contract_address`, otherwise can be null.

Reply:
For common, you can set `agent = consumer` when NOT use host contract. So, use `agent` as the chain-consumer, use `consumer` as the real-consumer.